mammoth/utils/statistics.py

```diff
@@ -148,13 +172,17 @@ def output(self, step, num_steps, learning_rate, start, metadata=None):
         meta_str = ''
         if num_steps > 0:
             step_fmt = "%s/%5d" % (step_fmt, num_steps)
+        acc = self.accuracy()
+        acc_str = f'{acc:6.2f}' if acc is not None else '--'
+        ppl = self.ppl()
+        ppl_str = f'{ppl:5.2f}' if ppl is not None else '--'
         logger.info(
-            ("%s: Step %s; acc: %6.2f; ppl: %5.2f; xent: %4.2f; %3.0f/%3.0f tok/s; %6.0f sec")
```

Comment:
convert to f-string, this is not readable

Reply:
This is old.